### PR TITLE
added Web Story discovery guide

### PR DESCRIPTION
**Pull request by CrystalOnScript**
screencapture-localhost-8080-documentation-guides-and-tutorials-optimize-and-measure-discovery-web-stories-2020-09-30-15_46_29

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+A URL to an image used as the [Web Story poster](https://amp.dev/documentation/components/amp-story/#poster-guidelines-(for-poster-portrait-src,-poster-landscape-src,-and-poster-square-src)). Used as the cover for the Web Story and should be representative of the story. Do not pre-embed the Web Story title on it. Should be 640 px by 640 px and maintain a 1:1 aspect ratio.
+
+
+### `supports-landscape`
+
+Allows a Web Story viewing on a mobile device held in a landscape orientation and changes the desktop experience to an immersive full-beed mode. Removes the default three portrait panel experience on desktop. 
+
+
+# HTML markup
```
Comment (pbakaus) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md:82]: I think these can also probably moved into the html metadata section, they technically all are

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+Maximize accessibility and indexability by including [meaningful alt-text for images](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Accessibility_concerns). 
+
+
+# Metadata
+
+Platforms that surface Web Stories rely on metadata to correctly index and display them. Include the following recommended information to give users a delightful Web Story experience. 
+
+## Schema.org metadata
+
+Including [schema.org](https://schema.org/) structured data vocabulary defines important information about your Web Story that allows third-party platforms, such as search engines, to display and index them.
```
Comment (pbakaus) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md:105]: Can we link to https://search.google.com/test/amp somewhere here? It's great for testing schema.org data.

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+## OGP metadata
+
+Including [Open Graph protocol](https://ogp.me/) enables Web Stories to become a rich object in a social graph.
```
Comment (pbakaus) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md:109]: I think it's fine calling out that this is important for Facebook.

**Review by pbakaus — CHANGES_REQUESTED** [review]
Cool, thanks! Left a bunch of comments.

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+
+### `publisher-logo-src`
+
+A URL to the Web Story publisher's logo image. The logo image should be larger than or equal to 96px by 96px and maintain a 1:1 aspect ratio.
+
+
+### `poster-portrait-src`
+
+A URL to an image used as the [Web Story poster](https://amp.dev/documentation/components/amp-story/#poster-guidelines-(for-poster-portrait-src,-poster-landscape-src,-and-poster-square-src)). Used as the cover for the Web Story and should be representative of the story. Do not pre-embed the Web Story title on it. Should be 853px by 640px and maintain a 3:4 aspect ratio.
```
[the remark this answers is not in the document]
Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md:63]: No problem, I'll update the language - but just an FYI, pre-embed is a word I copied from the public-facing spread sheet

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+## favicon
+
+Include a [favicon](https://www.w3.org/2005/10/howto-favicon) to display as the icon in the browser tab.
+
+## Web Story title
+
+Give your Web Story a title by including a <code>[<title>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title)</code> tag.
+
+## Image alt-text
```
[the remark this answers is not in the document]
Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md:94]: Are those metadata or a11y features?

[the remark this answers is not in the document]
Comment (pbakaus) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md:85]: Both :)

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (pbakaus) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/discovery_web_stories.md]: ty! Should ping the spreadsheet too :)

**Review by pbakaus — APPROVED** [review]

**Pull request merged**